primitives/src/authorization.rs

```diff
@@ -25,7 +25,7 @@ use sp_std::prelude::*;
 /// Authorization data for two step processes.
 #[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, PartialOrd, Ord)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub enum AuthorizationData<AccountId> {
+pub enum AuthorizationData<AccountId, Balance> {
```

Comment:
I'm thinking we should try to de-generify `Balance` from the code base, ending up with just `polymesh_primitives::Balance`. But this can be a separate task, and can be rolled out incrementally.

Reply:
That would make things a lot simpler.  I didn't like how many changed were needed to generalize `Balance`.